### PR TITLE
links updated in documentation

### PR DESCRIPTION
**Pull request by Mandeep-009**
this PR is in reference to the issue #4721

### CODE REVIEW

**Review by Jean-BaptisteC — APPROVED** [review]

File: docs/TRANSLATIONS.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Main:
 Additional:
 - Text-to-speech strings for navigation: [`data/strings/sound.txt`](../data/strings/sound.txt)
 
-- Android stores description: [`android/src/google/play/listings/`](../android/src/google/play/listings/)
+- Android stores description: [`android/src/google/play/listings/`](../android/src/google/play/)
```
Comment (biodranik) [inline docs/TRANSLATIONS.md:28]:
```suggestion
- Android stores description: [`android/src/google/play/`](../android/src/google/play/)
```

Comment (Mandeep-009) [reply docs/TRANSLATIONS.md:28]: @biodranik  but there does not exist any folder named "listings" in  google/play/

Comment (biodranik) [reply docs/TRANSLATIONS.md:28]: Exactly, so that's why I suggest to remove it.

**Review by biodranik — APPROVED** [review]
Thank you!

**Comment by biodranik** [comment]
Are all issues mentioned in #4721 fixed? If yes, then the issue can be closed.

**Comment by Mandeep-009** [comment]
yes, all the issues mentioned in #4721 were fixed.

**Comment by Jean-BaptisteC** [comment]
Not yet somes links in obj-c style redirect users to documentation archive

**Pull request merged**